api-authorization/server/authorization-middleware.js

```diff
@@ -0,0 +1,34 @@
+const jwt = require('jsonwebtoken'); // eslint-disable-line
+const ClientError = require('./client-error'); // eslint-disable-line
+
+function authorizationMiddleware(req, res, next) {
+  try {
+    const token = req.get('x-Access-Token');
+    if (!token) {
+      throw new ClientError(401, 'authentication required');
+    }
+    const payload = jwt.verify(token, process.env.TOKEN_SECRET);
+    req.user = payload;
+    next();
+  } catch (err) {
+    console.error(err);
+  }
```

Comment:
Any time you catch an error in an express application, you should pass it to `next()`